core/src/main/scala/org/apache/spark/shuffle/hash/HashShuffleReader.scala

```diff
@@ -17,10 +17,11 @@
 
 package org.apache.spark.shuffle.hash
 
-import org.apache.spark.{InterruptibleIterator, TaskContext}
 import org.apache.spark.serializer.Serializer
 import org.apache.spark.shuffle.{BaseShuffleHandle, ShuffleReader}
+import org.apache.spark.util.CompletionIterator
 import org.apache.spark.util.collection.ExternalSorter
+import org.apache.spark.{InterruptibleIterator, SparkEnv, TaskContext}
```

Comment:
Will do.

Btw, in an effort to keep from burdening reviewers with these style issues, I updated my Intellij environment to `Organize Imports...` [according to the Databrick style guide](https://github.com/databricks/scala-style-guide#imports) which I used to format this code. Here's my Intellij configuration for imports...

![screen shot 2015-06-09 at 3 44 42 pm](https://cloud.githubusercontent.com/assets/19554/8071406/86bb12f2-0ebe-11e5-9056-f031551f5019.png)

Do you or someone else have an Intellij configuration that works? We might want to update the style guide too.